crates/services/block_aggregator_api/src/blocks/old_block_source/convertor_adapter.rs

```diff
@@ -8,22 +15,18 @@ use crate::{
 };
 #[cfg(feature = "fault-proving")]
 use fuel_core_types::fuel_types::ChainId;
-
-use crate::blocks::old_block_source::convertor_adapter::fuel_to_proto_conversions::{
-    proto_header_from_header,
-    proto_receipt_from_receipt,
-    proto_tx_from_tx,
-};
 use fuel_core_types::{
     blockchain::block::Block as FuelBlock,
     fuel_tx::Receipt as FuelReceipt,
 };
+use prost::Message;
+use std::sync::Arc;
 
 #[derive(Clone)]
-pub struct ConvertorAdapter;
+pub struct ProtobufBlockConverter;
```

Comment:
We discussed the possibility of adding a large `Vec` here to use as a buffer that doesn't need to be reallocated each time.

Idk what the actual performance changes would be. But might be good.